iceberg/iceberg-handler/src/test/queries/positive/delete_iceberg_mixed.q

```diff
@@ -6,6 +6,8 @@
 --! qt:replace:/("removed-files-size":")\d+/$1#FileSize#/
 
 -- create an unpartitioned table with skip delete data set to false
+set hive.cbo.fallback.strategy=NEVER;
```

Comment:
How are the changes related to this test?

Reply:
This test falls back to non-cbo path due to `where null`:
```
[2024-06-06T22:35:22.382Z] delete from ice01 where null 
[2024-06-06T22:35:22.382Z] fname=delete_iceberg_mixed.q
[2024-06-06T22:35:22.382Z] 
[2024-06-06T22:35:22.382Z] See ./ql/target/tmp/log/hive.log or ./itests/qtest/target/tmp/log/hive.log, or check ./ql/target/surefire-reports or ./itests/qtest/target/surefire-reports/ for specific test cases logs.
[2024-06-06T22:35:22.382Z]  org.apache.hadoop.hive.ql.optimizer.calcite.CalciteSemanticException: Filter expression with non-boolean return type.
[2024-06-06T22:35:22.382Z] 	at org.apache.hadoop.hive.ql.parse.CalcitePlanner$CalcitePlannerAction.genFilterRelNode(CalcitePlanner.java:3212)
[2024-06-06T22:35:22.382Z] 	at org.apache.hadoop.hive.ql.parse.CalcitePlanner$CalcitePlannerAction.genFilterRelNode(CalcitePlanner.java:3198)
[2024-06-06T22:35:22.382Z] 	at org.apache.hadoop.hive.ql.parse.CalcitePlanner$CalcitePlannerAction.genFilterRelNode(CalcitePlanner.java:3395)
```